aspnet/fundamentals/logging.rst

```diff
@@ -16,23 +16,46 @@ Implementing Logging in your Application
 
-.. literalinclude:: logging/sample/src/LoggingSample/Startup.cs
+.. literalinclude:: logging/sample/src/TodoApi/Startup.cs
 	:language: c#
 	:linenos:
-	:lines: 17-27
+	:lines: 17-28
 	:dedent: 8
 	:emphasize-lines: 2,8-9
 
-When a logger is created, a category name or source must be provided. By convention this string is hierarchical, with categories separated by dot (``.``) characters. Some logging providers have filtering support that leverages this convention, making it easier to locate logging output of interest. In the above example, the logging is configured to use the built-in `ConsoleLogger <https://github.com/aspnet/Logging/blob/1.0.0-beta6/src/Microsoft.Framework.Logging.Console/ConsoleLogger.cs>`_ (see `Configuring Logging in your Application`_ below). To see the console logger in action, run the sample application using the ``web`` command, and make a request to configured URL (``localhost:5000``). You should see output similar to the following:
+When a logger is created, a category name or source must be provided. By convention this string is hierarchical, with categories separated by dot (``.``) characters. Some logging providers have filtering support that leverages this convention, making it easier to locate logging output of interest. In the above example, the logging is configured to use the built-in `ConsoleLogger <https://github.com/aspnet/Logging/blob/1.0.0-beta7/src/Microsoft.Framework.Logging.Console/ConsoleLogger.cs>`_ (see `Configuring Logging in your Application`_ below). To see the console logger in action, run the sample application using the ``web`` command, and make a request to configured URL (``localhost:5000``). You should see output similar to the following:
 
 .. image:: logging/_static/console-logger-output.png
 
 You may see more than one log statement per web request you make in your browser, since most browsers will make multiple requests (i.e. for the favicon file) when attempting to load a page. Note that the console logger displayed the log level (``info`` in the image above) followed by the category (``[LoggingSample.Startup]``), and then the message that was logged.
 
+In a real application, you will want to add logging based on application-level, not framework-level, events. For instance, if you have created a Web API application for managing To-Do Items (see `Building Your First Web API with MVC 6 <http://docs.asp.net/projects/mvc/en/latest/getting-started/first-web-api.html>`_), you could add logging around the various operations that can be performed on these items. To run this example, choose the ``mvcweb`` option in the Visual Studio run option selector. This option sets the environment to ``Mvc``, resulting in the use of the ``StartupMvc`` class, shown here:
```

Comment:
> In a real application, you will want to add logging based on application-level, not framework-level, events

How about: "Logging is generally used to log application-level events. Logging framework-level events (like logging when a request is received or a route is hit) is generally handled for you by the underlying frameworks."